deployments/ecmps/release-v2.0-fix/Sprint8/5995/camdecmpswks/copy_monitor_plan_to_workspace.sql

```diff
@@ -0,0 +1,339 @@
+-- PROCEDURE: camdecmpswks.copy_monitor_plan_to_workspace(text, text[])
+
+-- DROP PROCEDURE IF EXISTS camdecmpswks.copy_monitor_plan_to_workspace(text, text[]);
```

Comment:
In this version, we need to make sure the old version of the procedure is dropped.